setup.cfg

```diff
@@ -144,6 +144,10 @@ filterwarnings =
     # Ignore a warning we emit about not supporting the parallel
     # reading option for now, can be removed once the issue is fixed
     ignore:parallel reading does not currently work, so falling back to serial
+    # RuntimeWarning from convolution, lombscargle, modeling
+    ignore:invalid value encountered in true_divide:RuntimeWarning
+    # RuntimeWarning from units, lombscargle
+    ignore:divide by zero encountered in:RuntimeWarning
```

Comment:
I am not comfortable with blanket ignore like this. They should be ignored on a case-by-case basis.

Reply:
Yes, but it's more work, and at least for lombscargle I have no idea how to fix it.

Reply:
I still have a bad feeling about this. This will ignore such warnings for the entire repo, not just the affected subpackages.